.gitignore

```diff
@@ -224,6 +224,9 @@ bin/
 .cache-main
```

Comment:
Looks like the merge conflicts in this file haven't been resolved, the conflict symbols are still there.